scss/util/_color.scss

```diff
@@ -59,7 +59,7 @@
 /// @param {Color} $color2 - Second color to compare.
 ///
 /// @returns {Number} The contrast ratio of the compared colors.
-@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: 0) {
+@function color-pick-contrast($base, $colors: ($white, $black), $tolerance: $global-color-pick-contrast-tolerance) {
```

Comment:
@IamManchanda 
Thank you for your review 🍣 
I want to set default `tolerance` .

`color-pick-contrast` function is used by following components:
- button
- badge
- callout
- label
- tabs

When I use this function, I thought if `tolerance` can be used more flexible,
it'll be more convenient. For example, even Button's background is a bit light color,
I want use light color to font too.